tardis/montecarlo/setup_package.py

```diff
@@ -13,7 +13,19 @@ def get_extensions():
     sources += [os.path.relpath(fname) for fname in glob(
         os.path.join(os.path.dirname(__file__), 'src/randomkit', '*.c'))]
 
+    sources_test = ['tardis/montecarlo/tests/test_cmontecarlo.c']
+    sources_test += [os.path.relpath(fname) for fname in glob(
+        os.path.join(os.path.dirname(__file__), 'src', '*.c'))]
+    sources_test += [os.path.relpath(fname) for fname in glob(
+        os.path.join(os.path.dirname(__file__), 'src/randomkit', '*.c'))]
+
     return [Extension('tardis.montecarlo.montecarlo', sources,
                       include_dirs=['tardis/montecarlo/src',
                                     'tardis/montecarlo/src/randomkit',
+                                    'tardis/montecarlo/tests',
+                                    np.get_include()]),
+	    Extension('tardis.montecarlo.tests.test_cmontecarlo', sources_test,
```

Comment:
so in the end you should be able to do `import tardis.montecarlo.tests.test_cmontecarlo`